README.md

```diff
@@ -21,15 +21,110 @@ Bump is a Continuous Documentation Platform: it lets you keep your API doc alway
 
 ## Usage
 
-Start by creating a documentation on [Bump](https://bump.sh). Then add this workflow to your GitHub project:
+_Important: [actions/checkout](https://github.com/actions/checkout) has to be called **before this action**._
+
+Start by creating a documentation on [Bump](https://bump.sh). Then add one of the following workflow file to your GitHub project
+
+### API diff on pull requests
+
+If you only want to have API diff summary sent as a comment on your pull requests:
+
+`.github/workflows/bump.yml`
+
+```yaml
+name: API diff
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  api-diff:
+    name: Check API diff on Bump
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Comment pull request with API diff
+        uses: bump-sh/github-action@v1
+        with:
+          doc: <BUMP_DOC_ID>
+          token: ${{secrets.BUMP_TOKEN}}
+          file: doc/api-documentation.yml
+          command: diff
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+```
+
+_Important: make sure to change your main destination branch name (`main` in the example above) and replace `<BUMP_DOC_ID>` with your Bump documentation slug or id._
```

Comment:
Maybe we could also remember here that user should change file with his own path ?

Reply:
:+1: done